src/index.html

```diff
@@ -13,5 +13,151 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <label class="field">
+          Surname:
+          <input type="text" name="surname" autocomplete="off">
```

Comment:
если у тега больше двух атрибутов - то каждый атрибут пишем с новой строки
```suggestion
          <input
            type="text"
            name="surname"
            autocomplete="off"
          >
```